cot-cli/tests/snapshot_testing/cli/mod.rs

```diff
@@ -0,0 +1,43 @@
+use tempfile::TempDir;
+
+use super::*;
+#[test]
+fn no_args() {
+    insta::with_settings!({ filters => GENERIC_FILTERS.to_owned() },{ assert_cmd_snapshot!(cot_cli!("cli")) });
+}
+
+#[test]
+fn manpages() {
+    let tempdir = TempDir::new().unwrap();
+    insta::with_settings!({ filters => [GENERIC_FILTERS, TEMP_PATH_FILTERS].concat() },{ assert_cmd_snapshot!(cot_cli!("cli", "manpages", "-o", tempdir.path())) });
+}
+
+#[test]
+fn completions_missing_shell() {
+    insta::with_settings!({ filters => [GENERIC_FILTERS, TEMP_PATH_FILTERS].concat() },{ assert_cmd_snapshot!(cot_cli!("cli", "completions")) });
+}
```

Comment:
These guys exceed our line width limit and miss spaces around punctuation.

Note that `cargo fmt` sometimes cannot format code inside macros. Please fix this manually.